docs/src/design-system/pills/pills-badges-usage.tsx

```diff
@@ -0,0 +1,175 @@
+import { Box, Container, Toolbar, Typography, useTheme } from '@mui/material';
```

Comment:
Is there any difference between `import {Typography} from '@mui/material'` and `import { Typography } from '@ui-kit-2022/components';`? I tested that they had the same effect on our typography font styles.

Reply:
Ah, I forgot there was on in the library, too. I took a look, and the Typography component from `@ui-kit-2022/components` is just a reexported version of the one from MaterialUI, so they are the same.